functions/index.md

```diff
@@ -0,0 +1,11 @@
+# Get started with PostgreSQL functions
+TimescaleDB is built on top of PostgreSQL. When you interact with TimescaleDB,
+you're usually using PostgreSQL functions. This guide helps you get up and
+running with the most common PostgreSQL functions that you need to use with
+TimescaleDB.
+
+* [`INSERT`][insert] to insert data into a hypertable
+* [`ORDER BY`][order-by] to order the results of your query
```

Comment:
I don't understand why these are the two things we've decided to add here?

Reply:
Oh, it's just the beginning of the index. This will get built out as we add more sub pages.